src/content-handlers/iiif/extensions/uv-mediaelement-extension/Extension.ts

```diff
@@ -276,7 +276,14 @@ export default class Extension
       const annotations: Annotation[] = canvas.getContent();
 
       if (annotations && annotations.length) {
-        posterUri = annotations[0].getProperty("thumbnail");
+        let annotationUri = annotations[0].getProperty("thumbnail");
+        if (typeof annotationUri === "string") {
+          posterUri = annotationUri;
+        } else {
+          if (annotationUri.length && typeof annotationUri[0].id === "string") {
+            posterUri = annotationUri[0].id;
+          }
+        }
```

Comment:
Minor style suggestion:

```suggestion
        } else if (annotationUri.length && typeof annotationUri[0].id === "string") {
          posterUri = annotationUri[0].id;
        }
```